### PR TITLE
Update flexbox-axes.md

### PR DESCRIPTION
**Pull request by Warmastr**
This helped me wrap my mind around the code and I think it would provided a visual aid to reinforce what the student is learning.

 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

Added 'Additional Resources' section to Flexbox Axes section to visually reinforce the written material. The video author really explains axes in detail and provides useful examples that are beneficial to understanding the lesson. 

#### 2. Related Issue

Closes #XXXXX

### CODE REVIEW

File: foundations/html_css/flexbox/flexbox-axes.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ We've strayed from the point slightly... We were talking about flex-direction an
 
 > There are situations where the behavior of flex-direction could change if you are using a language that is written top-to-bottom or right-to-left, but you should save worrying about that until you are ready to start making a website in Arabic or Hebrew.
 
+### Additional Resources
+If you would like a good visual overview of flexboxes and how they can be used, check out the following video: <a href="https://www.youtube.com/watch?v=u044iM9xsWU">Flexboxes Made Simple</a> by Kevin Powell.
```
Comment (xandora) [inline foundations/html_css/flexbox/flexbox-axes.md:42]: Links should use markdown syntax, not HTML.

**Comment by xandora** [comment]
Is there a specific section of this video that learners should watch? The lesson is about flexbox axes, but since the video is 30+ minutes long, it seems to cover most of flexbox from what I can tell.

**Comment by Warmastr** [comment]
> Is there a specific section of this video that learners should watch? The lesson is about flexbox axes, but since the video is 30+ minutes long, it seems to cover most of flexbox from what I can tell.

Absolutely and I don't know why I didn't think to throw in a timestamp but that timestamp is from:

14:50
https://youtu.be/u044iM9xsWU?t=889 

to 23:59

**Comment by zachmmeyer** [comment]
Closing: Rejected.

Thanks for submitting this PR. Further discussion was had and the decision has been made to not include this content for this lesson at this current time. Please do feel free to contribute further in the future.

**Pull request closed without merging**